sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.net461.cs

```diff
@@ -3,7 +3,7 @@ namespace Azure
     public partial class CloudMachine
     {
         public CloudMachine(System.IO.Stream configurationContent) { }
-        public CloudMachine(string configurationFile = "cloudmachine.json") { }
+        public CloudMachine(string? configurationFile = null) { }
```

Comment:
Curious: why not 

```
public CloudMachine(string configurationFile = ".azure\\cloudmachine.json")
```
?

Reply:
I want it to run on linux too and thought this is the cleanest way to do it.